rosetta-source/src/main/rosetta/legaldocumentation-csa-type.rosetta

```diff
@@ -1,8 +1,182 @@
 namespace cdm.legaldocumentation.csa : <"Credit support concepts: CSA, collateral, elections, initial margin, threshold, minimum transfer amount.">
 version "${project.version}"
 
+import cdm.base.math.*
+import cdm.base.datetime.*
+import cdm.base.datetime.daycount.*
+import cdm.base.staticdata.party.*
+import cdm.base.staticdata.asset.common.*
+import cdm.base.staticdata.identifier.*
+
+import cdm.observable.asset.*
+
+import cdm.product.asset.FloatingRate
+import cdm.product.asset.FloatingRateBase
+import cdm.product.common.settlement.*
+import cdm.product.template.MarginTypeEnum
+
+
+import cdm.legaldocumentation.common.*
+
+import cdm.mapping.config.*
+
+import cdm.event.common.* 
+
+
+/* collateral intereset calculation and transfer enumerations */
+/* enumerations - should go into a separate file */
```

Comment:
As per comment, enums should be moved to a separate file (same namespace)

Reply:
How do I do this (create a new file in the package hierarchy)  in the Rosetta environment?  I couldn't find any way to do it, and didn't see any documentation that helped me.  What am I missing?  Is there an obvious button or link that I'm missing?

I was hoping that someone who knew how to do this trivial change would be able to do it, rather than having me spend a long time in correspondence, but hey, it's too late for that.